sdk/formrecognizer/azure-ai-formrecognizer/azure/ai/formrecognizer/_models.py

```diff
@@ -8,7 +8,7 @@
 
 from enum import Enum
 from collections import namedtuple
-from ._generated.models import ModelInfo, Error
+from ._generated.v2021_09_30_preview.models import ModelInfo, Error
```

Comment:
Being more explicit when importing from generated models now that we have some identical names, but different shapes across api versions.